src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java

```diff
@@ -771,6 +771,14 @@ private AdsDataProvider() {
   }
 
   @DataProvider
+  public static Object[][] slotsSizes() {
+    return new Object[][]{
+        {"adtest", "SKY/300x600", "LEFT_SKYSCRAPER_2", new Dimension(300, 600)},
+        {"adtest", "SKY/300x600", "LEFT_SKYSCRAPER_3", new Dimension(300, 600)},
+    };
+  }
+
+  @DataProvider
   public static Object[][] testPad() {
     return new Object[][]{
         {"adtest-pad", "Adtest-pad_Wikia", 250},
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "adtest-pad" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)